DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -115,45 +98,26 @@
 	- go to Settings -> Editor -> Code style -> Java : set class and static names counts for import with * to 99
 	- for Android Studio, code formatting is usually done with Ctrl+Alt+L. For automatic formatting, it's recommended to use the plugin Save Actions (https://plugins.jetbrains.com/plugin/7642-save-actions)
 
+### If debug mode does not work.
```

Comment:
Now this is a subheading of `##Android Studio`. I suggest to add this as part of the Issues paragraph below